### PR TITLE
add option to modify decimal digits

### PR DESCRIPTION
**Pull request by Itay289**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -225,7 +225,10 @@
 		strict: false,
 
 		// Unit to use for formatting.
-		unit: ''
+		unit: '',
+
+		// Decimal digits for formatting.
+		decimalDigits: 2
```
Comment (julien-f) [inline index.js]: Is not `decimals` clear enough?

**Comment by julien-f** [comment]
Thank you!

Would you mind adding some tests?

**Comment by Itay289** [comment]
Thank you for the great solution.

**Comment by julien-f** [comment]
Thank you!

**Pull request merged**